File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/DepartureDTO.cs
namespace SafarnamaApplication.DTOs
{
    public class DepartureDTO
    {
        public int DepartureId { get; set; }
        public int? PackageId { get; set; }
        public DateOnly? DepartDate { get; set; }
        public DateOnly? EndDate { get; set; }
        public int? NoOfDays { get; set; }
    }
}

File: Java_Backend/E-Tour/src/main/java/dto/DepartureDTO.java
package dto;


import lombok.*;
import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class DepartureDTO {
    private Integer departureId;
    private LocalDate departureDate;
    private LocalDate endDate;
    private Integer noOfDays;
    
    private Integer packageId;  
}
